ts/sagas/startup/watchLogoutSaga.ts

```diff
@@ -50,16 +48,17 @@ export function* watchLogoutSaga(
         };
         yield put(logoutFailure(logoutError));
       }
-      // Force the login by expiring the session
-      // FIXME: possibly reset the navigation stack as the watcher of
-      // SESSION_EXPIRED will save the navigation state and later restore it
-      yield put(sessionExpired());
     } catch (error) {
       const logoutError = {
         error,
         options: action.payload
       };
       yield put(logoutFailure(logoutError));
     }
+    // If keepUserData is false, startApplicationInitialization is
+    // dispatched within the componentDidMount of IngressScreen
+    if (action.payload.keepUserData) {
```

Comment:
This code could be added in a **_finally_** block

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1402/commits/c58b79747c31f19598e99cea95b2df0c438cff09